adding_species_manuscript.tex

```diff
@@ -155,42 +155,43 @@ \section*{Abstract}
-but implementing these simulations can require substantial time and specialized knowledge.
+and the sophistication of inference and simulation software.
+However, implementing these simulations still requires substantial time and specialized knowledge.
 These challenges are especially pronounced for simulating less well-studied species,
 since it is not always clear what level of realism is sufficient
 to confidently answer a given question, or what information is required
 to produce simulations of that desired realism.
-\Stdpopsim is a community-developed tool that seeks to lower this barrier
+\Stdpopsim is a community-developed framework that seeks to lower this barrier
 by making it easy to simulate complex population genetic models using
 up-to-date information.
-The initial version of the \stdpopsim catalog contained information for 
-6 species, most of which are well-characterized model organisms.
-Here, we report on community-driven efforts to expand the catalog 
-more broadly across the tree of life, which now contains 21 species,
-with 25 demographic models and 37 genetic maps.
-The process of expanding the \stdpopsim catalog 
-to include more species through community engagement yielded many insights, 
-and we report on lessons learned for best practices in population genomic simulation.
-We discuss the elements of a population genomic
-simulation model, including the required input data, common pitfalls and major considerations,
-and describe how new species models can be integrated into \stdpopsim.
-We also introduce several major advances to the realism of \stdpopsim's simulation ability,
-including gene conversion and provision of species-specific genomic annotations.
-Together, these advances to \stdpopsim will strengthen efforts to use and develop
-simulation-based population genomic inference methods, with particular advances
-for non-model organisms, making them available, transparent, and accessible to everyone.
+The initial version of \stdpopsim focused on establishing this framework using 
+six well-characterized model species.
+Here, we report on updates made in the new release of \stdpopsim (version 0.2),
+which include substantial additions to simulation capabilities and a significant expansion of the species catalog.
+%Ilan: moved this up here, to leave the end of the abstract to the bigger ideas
+Features added to improve the realism of the simulated genomes include non-crossover recombination and provision of species-specific genomic annotations.
+Expansion of the species catalog involved community-driven efforts to increase coverage more broadly across the tree of life
+to 21 species, with 25 demographic models and 37 genetic maps.
+Our experience throughout this process was that people are indeed keen to put in the time and effort to include their study species,
+but that simple, clear guidance is vital.
+Our intention with this paper is in part to provide another learning
```

Comment:
I also removed "another", since it's not clear what other learning modalities we're referring to. It also kind-of clashes with "also".
> Consequently, our intention with this paper is also to provide a learning modality to meet that need, ...